Makefile

```diff
@@ -283,6 +284,11 @@ else
 endif
 	@-oc create -f deploy/integreatly-rhmi-cr.yml
 
+.PHONY: prepare-patch-release
+prepare-patch-release:
+	$(CONTAINER_ENGINE) pull quay.io/integreatly/delorean-cli:master
+	$(CONTAINER_ENGINE) run --rm -e KUBECONFIG=/kube.config -v "${HOME}/.kube/config":/kube.config:z -v "${HOME}/.delorean.yaml:/.delorean.yaml" quay.io/integreatly/delorean-cli:master delorean release openshift-ci-release --config /.delorean.yaml --version $(TAG)
```

Comment:
Note this command won;t work until https://github.com/integr8ly/delorean/pull/33 is merged.